python_modules/libraries/dagster-fivetran/dagster_fivetran/translator.py

```diff
@@ -20,6 +25,21 @@ class FivetranConnectorTableProps(NamedTuple):
     service: Optional[str]
 
 
+class FivetranConnectorScheduleType(Enum):
+    """Enum representing each schedule type for a connector in Fivetran's ontology."""
+
+    AUTO = "auto"
+    MANUAL = "manual"
+
+    @classmethod
+    def has_value(cls, value) -> bool:
+        return value in cls._value2member_map_
+
+    @classmethod
+    def values(cls) -> Sequence[str]:
+        return list(cls._value2member_map_.keys())
```

Comment:
highly surprised you need to add these

Reply:
It is possible to list and use `__contains__` with the symbolic names, but no functions support `_value2member_map_` in Enum. This is to avoid calling  `_value2member_map_` in the resource.

Reply:
I see - so this is entirely due to the difference in casing between key and value? In that case maybe we should just make them match and add a comment

Reply:
Updated `FivetranConnectorScheduleType` to subclass both `str` and `Enum` in [8e607df](https://github.com/dagster-io/dagster/pull/25911/commits/8e607df725da0d5acf1fa03be9ed5460c6086144). Callsites have been updated too.